CHANGES.md

```diff
@@ -17,11 +17,12 @@ aligning related proposals, partly motivating a new Screens interface.
   - User-friendly screen labels, which vastly improve custom multi-screen picker and configuration UIs for apps
   - Screen change events, including changes to the multi-screen bit, to obviate polling
 - Chrome Security requested that this feature be disabled by default on embedded pages via a [permissions policy](https://w3c.github.io/webappsec-permissions-policy/)
-  - In the second origin trial, cross origin iframes need to specify an `allow="window-placement"` attribute.
+  - In the second origin trial, cross origin iframes need to specify an `allow="window-management"` attribute.
```

Comment:
Combine this statement with the one above and avoid specifying the string inline:
- Starting in the second origin trial, a [permissions policy](https://w3c.github.io/webappsec-permissions-policy/) was added, so embedders could control feature access of embedded iframes.

Reply:
Done